examples/jsm/nodes/accessors/CubeTextureNode.js

```diff
@@ -0,0 +1,100 @@
+import UniformNode from '../core/UniformNode.js';
+import ReflectNode from './ReflectNode.js';
+
+class CubeTextureNode extends UniformNode {
```

Comment:
Maybe it would better to make CubeTextureNode extend TextureNode?

Reply:
Yes, good catch!